### PR TITLE
Major upgrade of the code to performe an automated unfolding proceedu…

### PR DESCRIPTION
**Pull request by efranzis**
…re with QA checks in between

@jdmulligan take your time and go through the changes. Maybe we can start going through 
the code today. 
Please start by looking at 
- loadYamlAndRun.py
and
unfoldJetSpectrum.py

### CODE REVIEW

File: analyses/dijetImbalance/loadYamlAndRun.py [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+# And which components of this script you want to run
+#
+# . . . . . . . . . . . .
+# To Note there are for now hard coded values in the single subscripts:
+# -A hard coded value of importance is the regularization parameter in the unfolding script you might want
+# to change -> depending on how your unfolding QA looks like
+# - In summarizeErrors you have the fit function to set
+# . . . . . . . . . . . .
+# To be implemented:
+#
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/loadYamlAndRun.py]: done

File: analyses/dijetImbalance/loadYamlAndRun.py [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+# To use this script you need a yaml file wich contains all the paths to the train output .root files
+# it should contain all the configurations starting from L68
+# Only thing which you have to additionally set is where your yaml files are and will be stored -> YamlFilePaths
+# And which components of this script you want to run
+#
+# . . . . . . . . . . . .
+# To Note there are for now hard coded values in the single subscripts:
+# -A hard coded value of importance is the regularization parameter in the unfolding script you might want
+# to change -> depending on how your unfolding QA looks like
+# - In summarizeErrors you have the fit function to set
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/loadYamlAndRun.py]: done

File: analyses/dijetImbalance/unfoldJetSpectrum.py [hunk 3]
```diff
@@ -147,93 +203,114 @@ def unfoldJetSpectrum(inputFileData, inputFileResponse, inputFileResponse1, inpu
...
-
-      # Plot comparison of measured spectrum from each list (for the moment, errors only work if I comment out above loop)
-      plotDetLevelRatio(ispp, fData, labelList, minCent, maxCent, minPtDet, maxPtDet, nBinsDet, detBinArray, outputDir, fileFormat)
+  plotJetResponse(ispp, fResponse, label, hUnfoldedResultDict, minCent, maxCent, minPtGen, maxPtGen, isClosureTest, outputDir, fileFormat)
 
-    # Plot metrics of jet response: JES shift, JER, etc. for each response matrix
-    compareJetResponse(fResponse, labelList, hUnfoldedResultDict, minCent, maxCent, minPtGen, maxPtGen, nBinsTruth, truthBinArray, outputDir, fileFormat)
-  
+  #- - - - - - - - - - - - - - - - - - - - - - - - - -
+  # Get the particle level pythia spectrum and save it to the results file too ELIANE
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/unfoldJetSpectrum.py]: Done - is OK

File: analyses/dijetImbalance/summarizeUnfolding.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python
+#
+# Plot ratio of unfolded spectrum from f2, f3 to f1
+#
+# To run:
+#     python plotCompareUnfoldedResults.py -a "/ref/file/AnalysisResults.root" -b /file2/AnalysisResults.root
+#     -c /file3/AnalysisResults.root -o /output/dir/" -i ".png"
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/summarizeUnfolding.py]: done

File: analyses/dijetImbalance/unfoldJetSpectrum.py [hunk 5]
```diff
@@ -1059,17 +1082,21 @@ def performRefoldingTest(ispp, response1, response2, hJetSpectrumMeasuredPerBin,
...
-  h2LegendLabel = "Measured Pb-Pb"
+
   ratioYAxisTitle = "Refolded / Measured"
   outputDirRefoldingTest = outputDir + "RefoldingTest/"
   if not os.path.exists(outputDirRefoldingTest):
     os.makedirs(outputDirRefoldingTest)
-  outputFilename = os.path.join(outputDirRefoldingTest, "hJetSpectraRefoldingTest{}_{}{}".format(type, i, fileFormat))
+  outputFilename = os.path.join(outputDirRefoldingTest, "hJetSpectraRefoldingTest_{}{}".format( i, fileFormat))
   plotSpectra(hFoldedPbPbTruth1PerGeV, hJetSpectrumMeasuredPerGeV, "", 1., xRangeMin, xRangeMax, yAxisTitle, ratioYAxisTitle, outputFilename, "", legendTitle, h1LegendLabel, h2LegendLabel)
+ # plotSpectra(hFoldedPbPbTruth1PerGeV, "", "", 1., xRangeMin, xRangeMax, yAxisTitle, ratioYAxisTitle, outputFilename, "", legendTitle, h1LegendLabel, h2LegendLabel)
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/unfoldJetSpectrum.py]: deleted

File: analyses/dijetImbalance/plotRaa.py [hunk 6]
```diff
@@ -60,324 +67,365 @@ def plotRaa(fUnfoldedResult, fJetRecoEfficiency, fppResult, fPowheg, outputDir,
...
+  if plotPbPb:
+    print("find PbPb data histogram: hJetSpectrumUnfoldedPerGeV{}_".format(label))
+    hDataPbPb = fPbPbfile.Get("hJetSpectrumUnfoldedPerGeV{}_".format(label))
+    hDataPbPb.SetName("hDataPbPb")
+    outputFilename = os.path.join(outputDir, "PbPb_Input" + fileFormat)
+    plotHist(hDataPbPb, outputFilename,"E",True)
+    hPbPbJetSpectrum = RebinHisto(hDataPbPb, hDataPbPb, minPtReported, maxPtReported)
+
+    # Get the jet reconstruction efficiency
+    hJetRecoEffPbPb  = fPbPbfile.Get("hEfficiency")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: The Pb-Pb case needs to be able to accept a different jet reconstruction efficiency, namely we use the pp efficiency -- that's why plotRaa takes in a separate file for the jet reconstruction efficiency

File: analyses/dijetImbalance/plotRaa.py [hunk 7]
```diff
@@ -60,324 +67,365 @@ def plotRaa(fUnfoldedResult, fJetRecoEfficiency, fppResult, fPowheg, outputDir,
...
-    plotJetRaa(RAAStat, RAACorr, RAAShape, label, color, colorLight, outputDir, fileFormat)
+    print("find PbPb data histogram: hJetSpectrumUnfoldedPerGeV{}_".format(label))
+    hDataPbPb = fPbPbfile.Get("hJetSpectrumUnfoldedPerGeV{}_".format(label))
+    hDataPbPb.SetName("hDataPbPb")
+    outputFilename = os.path.join(outputDir, "PbPb_Input" + fileFormat)
+    plotHist(hDataPbPb, outputFilename,"E",True)
+    hPbPbJetSpectrum = RebinHisto(hDataPbPb, hDataPbPb, minPtReported, maxPtReported)
+
+    # Get the jet reconstruction efficiency
+    hJetRecoEffPbPb  = fPbPbfile.Get("hEfficiency")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: same here

File: analyses/dijetImbalance/plotRaa.py [hunk 8]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
     jetSpectrumPPShape.SetFillColor(600-6) #920
     jetSpectrumPPShape.SetFillColorAlpha(600-6, 0.3)
     jetSpectrumPPShape.SetFillStyle(1001)
-
   if plotPbPb:
     jetSpectrumPbPbCorr.SetFillStyle(0)
     jetSpectrumPbPbCorr.SetLineColor(color)
     jetSpectrumPbPbShape.SetLineColor(0)
-    jetSpectrumPbPbShape.SetFillColor(colorLight)
+    jetSpectrumPbPbShape.SetFillColor(color)
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: Why the color changes?

File: analyses/dijetImbalance/plotRaa.py [hunk 9]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
+    jetSpectrumPbPbShape.SetFillColorAlpha(color, 0.3)
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: and here?

File: analyses/dijetImbalance/plotRaa.py [hunk 10]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
   system2.SetNDC()
-  system2.SetTextSize(0.044)
+  system2.SetTextSize(0.044*scale)
   system2.Draw()
 
   if "02" in label:
-    myLegend = ROOT.TLegend(0.52,0.7,0.7,0.82)
-    setupLegend(myLegend, 0.035) #0.04
+    myLegend = ROOT.TLegend(0.26+labelOffset1,0.7,0.7,0.82)
+    setupLegend(myLegend, 0.035*scale*1.1) #0.04
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: I am having a hard time if these things are changed from before or not, let's discuss

File: analyses/dijetImbalance/plotRaa.py [hunk 11]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
-    myLegend = ROOT.TLegend(0.18+labelOffset1,0.69,0.5,0.81)
-    setupLegend(myLegend, 0.035) #0.04
+    myLegend = ROOT.TLegend(0.26+labelOffset1,0.7,0.7,0.82)
+    setupLegend(myLegend, 0.035*scale*1.1) #0.04
     myLegend.AddEntry(myBlankHisto,"Anti-#it{k}_{T} #it{R} = 0.4  | #it{#eta}_{jet}| < 0.3","")
-  if plotPbPb:
-    if "5GeV" in label:
-      myLegend.AddEntry(myBlankHisto,"#it{p}_{T}^{lead,ch} > 5 GeV/#it{c}","")
-    elif "7GeV" in label:
-      myLegend.AddEntry(myBlankHisto,"#it{p}_{T}^{lead,ch} > 7 GeV/#it{c}","")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: Is the 7 GeV case removed now? We still need it

File: analyses/dijetImbalance/plotRaa.py [hunk 12]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
-    myLegend3bis.AddEntry(jetSpectrumPowhegScale,"Scale uncertainty","f")
-    myLegend3bis.AddEntry(jetSpectrumPowhegPdf,"PDF uncertainty","f")
+    myLegend3bis = ROOT.TLegend(0.25,0.18+shift,0.65,0.27+shift)
+    setupLegend(myLegend3bis, 0.031*scale)
+    myLegend3bis.AddEntry(jetSpectrumPbPbCorr,"Correlated uncertainty","f")
+    myLegend3bis.AddEntry(jetSpectrumPbPbShape,"Shape uncertainty","f")
     myLegend3bis.Draw()
 
-  if plotPbPb:
-    sysTAA = ROOT.TLatex(0.65,0.26,"#sigma_{#it{T}_{AA}}^{sys} = 3.3%")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: why is this removed?

File: analyses/dijetImbalance/plotRaa.py [hunk 13]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
+
+  # Create canvas
+  cSpectrum = ROOT.TCanvas("cSpectrumRatio{}","cSpectrumRatio: hist",800,850)
+  cSpectrum.Draw()
+  cSpectrum.cd()
+
+  # Set pad and histo arrangement
   myPad = ROOT.TPad("myPad", "The pad",0,0,1,1)
-  myPad.SetMargin(0.18,0.04,0.15,0.04)
+  myPad.SetLeftMargin(0.17)
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: why is it changed?

File: analyses/dijetImbalance/plotRaa.py [hunk 14]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
-  myBlankHisto.Draw()
-  
-  RAAStat.SetMarkerSize(1.5)
-  RAAStat.SetMarkerStyle(21)
-  RAAStat.SetMarkerColor(color)
-  RAAStat.SetLineStyle(1)
-  RAAStat.SetLineWidth(2)
-  RAAStat.SetLineColor(color)
+  myBlankHisto.SetXTitle("#it{p}_{T,jet} (GeV/#it{c})")
+  myBlankHisto.GetYaxis().SetTitleOffset(1.8)
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: why is it changed?

File: analyses/dijetImbalance/plotRaa.py [hunk 15]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
   
-  RAAShape.Draw("E2Same")
-  RAACorr.Draw("E2Same")
-  RAAStat.Draw("Z P same")
+  #Create a histrogram with same bin entries as main histogram
+  spectrumRAACorrErr  = jetSpectrumPbPbStat.Clone("RAACorrErrors")
+  spectrumRAAShapeErr = jetSpectrumPbPbStat.Clone("RAAShapeErrors")
+
+  #propagate the error by building the ratio of the error histograms
+  jetSpectrumPbPbCorr.Divide(jetSpectrumPPCorr)
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: is this right?

File: analyses/dijetImbalance/plotRaa.py [hunk 16]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
+  spectrumRAAShapeErr.Draw("E2 same")
+  spectrumRAACorrErr.Draw("E2 same")
+  gJetRAA.Draw("Z P same")
+
+  # # # # # # # # # # # # # # # # # # # # # # # #
+  # Add legends and text
+  # # # # # # # # # # # # # # # # # # # # # # # #
+  labelOffset1=+0.02
   
-  system2 = ROOT.TLatex(0.23,0.89,"ALICE Preliminary")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: why is this changed?

File: analyses/dijetImbalance/plotRaa.py [hunk 17]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
-
-  myLegend2 = ROOT.TLegend(0.25,0.5,0.5,0.57)
-  setupLegend(myLegend2,0.033)
-  myLegend2.SetNColumns(1)
-  myLegend2.AddEntry(RAAStat," 0 - 10%","Pe")
-  myLegend2.Draw()
+    myLegend = ROOT.TLegend(0.03,0.66,0.7,0.78)
+    setupLegend(myLegend, 0.035*scale*1.1) #0.04
+    myLegend.AddEntry(myBlankHisto,"Anti-#it{k}_{T} #it{R} = 0.4  | #it{#eta}_{jet}| < 0.3","")
+  myLegend.AddEntry(myBlankHisto,"#it{p}_{T}^{lead,ch} > 5 GeV/#it{c}","")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: Need 7 GeV capability

File: analyses/dijetImbalance/plotRaa.py [hunk 18]
```diff
@@ -386,311 +434,471 @@ def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShap
...
+  myLegend.Draw()
 
-  myLegend3bis = ROOT.TLegend(0.25,0.41,0.5,0.51)
-  setupLegend(myLegend3bis,0.027)
-  myLegend3bis.AddEntry(RAACorr," Correlated uncertainty","f")
-  myLegend3bis.AddEntry(RAAShape," Shape uncertainty","f")
+  myLegend2 = ROOT.TLegend(0.19,0.52,0.45,0.6)
+  setupLegend(myLegend2,0.035*scale)
+  #myLegend2.AddEntry(gJetRAA,"10% Pb-Pb","Pe")
+  myLegend2.AddEntry(gJetRAA,"ALICE Data","Pe")
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: Somewhere needs to say 0-10%

File: analyses/dijetImbalance/plotRaa.py [hunk 19]
```diff
@@ -776,80 +983,39 @@ def setOptions():
...
-def RebinDataHisto(h, minPtReported, maxPtReported, nBinsTruth, truthBinArray):
-  
-  hNew = ROOT.TH1D("hNew{}".format(h.GetName()), "hNew{}".format(h.GetName()), nBinsTruth, truthBinArray)
-  hNew.Sumw2()
+def RebinHisto(hMainData, hInput, minPtReported, maxPtReported, binErr=-1):
   
-  startBin = h.FindBin(minPtReported + 1.)
+  #Get data historgam properties
+  nBinsTruth= hMainData.GetXaxis().GetNbins()
+  startBin  = hMainData.FindBin(minPtReported)
```
Comment (jdmulligan) [inline analyses/dijetImbalance/plotRaa.py]: careful about bin edge

File: analyses/dijetImbalance/plotRaa.py (hunk 9 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: Because you don't need to define a new color for this. You can simply set the SetFillColorAlpha

File: analyses/dijetImbalance/plotRaa.py (hunk 10 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: This needed to get changed a bit because I added the case that not only the spectra are plotted but also the ration to the theory. This you see in L433-438

File: analyses/dijetImbalance/plotRaa.py (hunk 11 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: changed it to be set automatically from the label string now

File: analyses/dijetImbalance/plotRaa.py (hunk 12 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: I honestly think I had an older version I was working with and overwriting. I will put it in now. Sorry.

File: analyses/dijetImbalance/plotRaa.py (hunk 17 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: read now from the label automatically

File: analyses/dijetImbalance/plotRaa.py (hunk 18 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: Isn't this in L850?

File: analyses/dijetImbalance/plotRaa.py (hunk 19 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: I added now a 0.1 to the minPtReported

File: analyses/dijetImbalance/summarizeUnfolding.py [hunk 20]
```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env python
+#
+# This script plots a powerpoint presentation that
+# summarizes all the unfolding QA so that you can quickly check
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/summarizeUnfolding.py]: done

File: analyses/dijetImbalance/summarizeErrors.py [hunk 21]
```diff
@@ -0,0 +1,735 @@
+#! /usr/bin/env python
+#
+# Plot ratio of unfolded spectrum from f2, f3 to f1
+#
+# To run:
+#     python plotCompareUnfoldedResults.py -a "/ref/file/AnalysisResults.root" -b /file2/AnalysisResults.root
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/summarizeErrors.py]: done

File: analyses/dijetImbalance/summarizeErrors.py [hunk 22]
```diff
@@ -0,0 +1,735 @@
...
+    
+  #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  #..Correlated Errors
+  hTrack   = fTrack.Get("hNewh2")
+  hTrack.SetName("hTrack")
+  hTrack.SetLineColor(kGreen-6)
+  hHadCorr = fHadCorr.Get("hNewh2")
+  hHadCorr.SetName("hHadCorr")
+  hHadCorr.SetLineColor(kCyan-6)  #ELIANE: something is wrong with this
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/summarizeErrors.py]: done

File: analyses/dijetImbalance/summarizeErrors.py [hunk 23]
```diff
@@ -0,0 +1,735 @@
...
+  hMethod.GetXaxis().SetTitle("unfolded #it{p}_{T}^{jet} (GeV/#it{c})")
+  if plotOrig==0:
+    hMethod.GetYaxis().SetTitle("systematic difference in %")
+    hMethod.GetYaxis().SetRangeUser(0,10)
+    hMethod.GetYaxis().SetRangeUser(0,15)
+  else:
+    hMethod.GetYaxis().SetTitle("absolute difference of spectra")
+
+  hMethod.Draw("hist")
+  averRange.Draw("same hist")
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/summarizeErrors.py]: done changed all histograms to begin with a h

File: analyses/dijetImbalance/plotRaa.py [hunk 24]
```diff
@@ -60,324 +67,365 @@ def plotRaa(fUnfoldedResult, fJetRecoEfficiency, fppResult, fPowheg, outputDir,
...
+      perErr = hPercError.GetBinError(bin)
+      hSpectrum.SetBinError(bin,perErr)
 
 ########################################################################################################
 # Plot PbPb jet spectrum ###############################################################################
 # Copied heavily from fig_PbPbSpectraCentR02Bias5NEW.C from 2.76 TeV Full jet analysis      ############
 ########################################################################################################
-def plotJetSpectra(jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShape, jetSpectrumPPStat, jetSpectrumPPCorr, jetSpectrumPPShape, plotPbPb, plotPP, plotPowheg, jetSpectrumPowhegStat, jetSpectrumPowhegPdf, jetSpectrumPowhegScale, label, color, colorLight, outputDir, fileFormat):
-
+def plotJetSpectra(plotRatioToo, plotPP, jetSpectrumPPStat, jetSpectrumPPCorr, jetSpectrumPPShape, plotPbPb, jetSpectrumPbPbStat, jetSpectrumPbPbCorr, jetSpectrumPbPbShape, plotPowheg, jetSpectrumPowhegStat, jetSpectrumPowhegPdf, jetSpectrumPowhegScale, pythiaSpectrum, label, color, colorLight, outputDir, fileFormat):
```
[the remark this answers is not in the document]
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: done

File: analyses/dijetImbalance/plotRaa.py (hunk 8 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: Because you don't need to define a new color for this. You can simply set the SetFillColorAlpha

File: analyses/dijetImbalance/plotRaa.py (hunk 7 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: Done. Now one can use the PbPb efficiency for testing purposes but it will print a warning to load a pp file as well.

File: analyses/dijetImbalance/plotRaa.py (hunk 6 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: Done. Now one can use the PbPb efficiency for testing purposes but it will print a warning to load a pp file as well.

File: analyses/dijetImbalance/plotRaa.py (hunk 16 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: I think that whatever figures we will do going forward will not have this label anymore

File: analyses/dijetImbalance/plotRaa.py (hunk 15 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: yes, I think we discussed this. Correct me if I am wrong

File: analyses/dijetImbalance/plotRaa.py (hunk 14 above)
Comment (efranzis) [reply analyses/dijetImbalance/plotRaa.py]: you mean the RAAStat settings? That's from L777 onward
